tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -237,8 +236,21 @@ sub install_kubevirt_packages {
     zypper_call('in jq open-iscsi') if (script_run('rpmquery jq open-iscsi') && ($kubevirt_ver ge "0.50.0"));
 
     # Install required packages perl-CPAN-Changes and ant-junit
+    my $repo_name = '';
+    my $inst_pkgs = '';
     if (is_transactional) {
-        $self->install_additional_pkgs();
+        if (get_var("INSTALL_OTHER_REPOS")) {
+            foreach (split(/,/, get_var("INSTALL_OTHER_REPOS"))) {
+                $repo_name = (split(/\//, $_))[-1] . "-" . bmwqemu::random_string(8);
+                zypper_call("--gpg-auto-import-keys ar --enable --refresh $_ $repo_name");
+                save_screenshot;
+            }
+            zypper_call("--gpg-auto-import-keys refresh");
+            save_screenshot;
+            $inst_pkgs = $inst_pkgs . " $_" foreach (split(/,/, get_required_var("INSTALL_OTHER_PACKAGES")));
+            zypper_call('in -f' . $inst_pkgs);
```

Comment:
Would you please also remove the repo after installing the assistant pkgs? BTW, a comment describing that this is temporary solution and will be removed after ant is not replaced by other way in roadmap 18, will be helpful. Thanks.

Reply:
SLM increments are failing by missing this. So merge it now. Please improvethis minor thing in new PR.